### PR TITLE
[Security] Fix resetting traceable listeners

### LINKED ISSUE #49532: TraceableAuthenticatorManagerListener OOM
### Symfony version(s) affected

6.1.*

### Description

I have a project using symfony and swoole runtime.
In development, the memory will not release after request finished.
After some research, we found https://github.com/BitOne/php-meminfo and use find that the \Symfony\Component\Security\Http\Authenticator\Debug\TraceableAuthenticatorManagerListener::$authenticatorsInfo will increase after request finished, then the process OOM.

### How to reproduce

1. Install symfony and swoole runtime or any other long-process runtime;
2. Login in user;
3. Refresh the page and watch the system memory refresh;

### Possible Solution

implement the ResetInterface and reset $authenticatorsInfo in reset function.

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #49532 
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/Security/Http/composer.json [hunk 1]
```diff
@@ -18,12 +18,13 @@
...
         "symfony/deprecation-contracts": "^2.1|^3",
-        "symfony/security-core": "^5.4.19|~6.0.19|~6.1.11|^6.2.5",
         "symfony/http-foundation": "^5.3|^6.0",
         "symfony/http-kernel": "^5.3|^6.0",
         "symfony/polyfill-mbstring": "~1.0",
         "symfony/polyfill-php80": "^1.16",
-        "symfony/property-access": "^4.4|^5.0|^6.0"
+        "symfony/property-access": "^4.4|^5.0|^6.0",
+        "symfony/service-contracts": "^1.10|^2|^3",
+        "symfony/security-core": "^5.4.19|~6.0.19|~6.1.11|^6.2.5"
```
Comment (OskarStark) [inline src/Symfony/Component/Security/Http/composer.json]: This line contains no diff, except the trailing `,`. Can you please move it back up? Thanks

File: src/Symfony/Component/Security/Http/composer.json [hunk 2]
```diff
@@ -18,12 +18,13 @@
     "require": {
         "php": ">=7.2.5",
         "symfony/deprecation-contracts": "^2.1|^3",
-        "symfony/security-core": "^5.4.19|~6.0.19|~6.1.11|^6.2.5",
         "symfony/http-foundation": "^5.3|^6.0",
         "symfony/http-kernel": "^5.3|^6.0",
         "symfony/polyfill-mbstring": "~1.0",
         "symfony/polyfill-php80": "^1.16",
-        "symfony/property-access": "^4.4|^5.0|^6.0"
+        "symfony/property-access": "^4.4|^5.0|^6.0",
```
Comment (OskarStark) [inline src/Symfony/Component/Security/Http/composer.json:25]: This line contains no diff, except the trailing `,`. Can you please move it back down? Thanks

Comment (nicolas-grekas) [reply src/Symfony/Component/Security/Http/composer.json:25]: not sure, this respects alpha ordering

File: src/Symfony/Component/Security/Http/composer.json (hunk 1 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/Security/Http/composer.json]: this one should be alpha-ordered

File: src/Symfony/Bundle/SecurityBundle/composer.json [hunk 3]
```diff
@@ -29,7 +29,8 @@
         "symfony/security-core": "^5.4|^6.0",
         "symfony/security-csrf": "^4.4|^5.0|^6.0",
         "symfony/security-guard": "^5.3",
-        "symfony/security-http": "^5.4.20|~6.0.20|~6.1.12|^6.2.6"
+        "symfony/security-http": "^5.4.30|~6.0.20|~6.1.12|^6.2.6|^6.3.6",
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/SecurityBundle/composer.json]:
```suggestion
        "symfony/security-http": "^5.4.30|^6.3.6",
```

**Comment by chalasr** [comment]
Comments addressed, thanks

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @chalasr.

**Pull request merged**